src/components/Swap/Toolbar/Toolbar.test.tsx

```diff
@@ -0,0 +1,72 @@
+import { TradeType } from '@uniswap/sdk-core'
+import { DAI, USDC_MAINNET } from 'constants/tokens'
+import { SwapInfoProvider } from 'hooks/swap/useSwapInfo'
+import { flagsAtom } from 'hooks/useSyncFlags'
+import Module from 'module'
+import { Field, stateAtom, Swap } from 'state/swap'
+import { renderComponent } from 'test'
+
+import Toolbar from './index'
+
+jest.mock('@web3-react/core', () => ({
+  ...(jest.requireActual('@web3-react/core') as Module),
+  useWeb3React: () => ({
+    isActive: true,
+    chainId: 1,
+    account: '0x0000000000000000000000000000000000000000',
+  }),
+}))
+
+jest.mock('../../../hooks/usePermit2Allowance', () => ({
```

Comment:
Can you specify the mocked file as `'hooks/usePermit2Allowance'` (absolute filepath) instead of using a relative filepath?

Reply:
yes!